Common/tests/AdapterLoading/AdapterLoadingTest.cs

```diff
@@ -58,7 +58,7 @@ public static IEnumerable TestCasesQueueLocation
     }
   }
 
-  public void Setup(Dictionary<string, string?> config)
+  public static void Setup(Dictionary<string, string?> config)
```

Comment:
Is this actually used? If so, how can it do anything at all?

Reply:
Yes it is used. I renamed it to give a more explicit name.